client/components/TestQueueRun/index.jsx

```diff
@@ -47,36 +47,16 @@ class TestQueueRow extends Component {
         }
     }
 
-    renderTestsCompletedByUser(uid) {
-        const { testsForRun, usersById } = this.props;
-
-        let totalTests = testsForRun.length;
-        let testsCompleted = testsForRun.filter(
-            t =>
-                t.results &&
-                t.results[uid] &&
-                t.results[uid].status === 'complete'
-        ).length;
+    testRun() {
```

Comment:
Nonblocking: This also might benefit from a renaming to `testRunAriaLabel`. I was a little confused by the naming `testRun` because it was generic.

Reply:
way better. will fix next time I'm here